common/src/logging.rs

```diff
@@ -3,7 +3,15 @@ use tracing_subscriber::layer::SubscriberExt;
 use tracing_subscriber::util::SubscriberInitExt;
 use tracing_subscriber::EnvFilter;
 
-pub fn setup_logging() {
+pub fn setup_logging(pretty: bool) {
+    if pretty {
+        setup_logging_pretty()
+    } else {
+        setup_logging_json()
+    }
```

Comment:
I personally prefer this, does anybody have an opinion?
```rust
    match pretty {
        true => setup_logging_pretty(),
        false => setup_logging_json(),
    }
```

Reply:
No preference.

Reply:
I like the match statement more. Will update.

Reply:
Done.